Dockerfile

```diff
@@ -0,0 +1,23 @@
+#Get Php 
+FROM php:7.1.1
+
+#Install composer php and mongodb
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv EA312927 \
+  && echo "deb http://repo.mongodb.org/apt/debian wheezy/mongodb-org/3.2 main" | tee /etc/apt/sources.list.d/mongodb-org-3.2.list \
```

Comment:
Is there a reason for deploying MongoDB within the Docker container? The purpose of the application is to connect to an existing MongoDB cluster to work with its `system.profile` collections. Having the application connect to its own MongoDB instance (with no data) doesn't have much value.